src/app/basics/exploration/why-time-travel/page.md

```diff
@@ -9,36 +9,36 @@ Problems in complex web applications often depend on particular network, timing,
 
 Replay is the first deterministic browser, so all bugs recorded with Replay are **reproducible**, no matter how complex or rare they are. This means your engineers **don’t have to spend days** trying to reproduce a flaky test or rare customer support issue.
 
-[Learn more](/replay-devtools/overview)
```

Comment:
I'd change exploration to introduction. This is important for redirects so lets get this in first.